src/components/Menu/ActionList.vue

```diff
@@ -0,0 +1,88 @@
+<!--
+  - @copyright Copyright (c) 2022 Vinicius Reis <vinicius@nextcloud.com>
+  -
+  - @author Vinicius Reis <vinicius@nextcloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Actions v-tooltip="tooltip"
+		class="entry-list-action entry-action"
+		v-bind="state"
+		:title="actionEntry.label"
+		:data-text-action-entry="actionEntry._key"
+		v-on="$listeners">
+		<template #icon>
+			<component :is="icon" />
```

Comment:
```suggestion
			<component :is="icon" :size="20" />
```

Fits better into the regular Nextcloud icons if the size is set to 20, this is also how other apps are doing for embedded material design icons it like Talk.

Reply:
I change the [`icons.js`](https://github.com/nextcloud/text/pull/2439/files#diff-c8b7366367478f7f700b8a5f4d3a363a306fb30907c4f1b240045d1daa828f0a) now, all icons will use `20px` as [default size](https://github.com/nextcloud/text/pull/2439/commits/13e2a0d66b45844acd44a8f7ae209eacabc3ed1b).